unicorn_mode/README.md

```diff
@@ -105,11 +104,8 @@ An example use of AFL-Unicorn mode is discussed in the paper Unicorefuzz:
 As for the QEMU-based instrumentation, unicornafl comes with a sub-instruction
 based instrumentation similar in purpose to laf-intel.
 
-The options that enable Unicorn CompareCoverage are the same used for QEMU. This
-will split up each multi-byte compare to give feedback for each correct byte:
-
-* `AFL_COMPCOV_LEVEL=1` to instrument comparisons with only immediate values.
-* `AFL_COMPCOV_LEVEL=2` to instrument all comparison instructions.
+`UNICORN_AFL_CMPCOV=1` will split up each multi-byte compare to give feedback for 
```

Comment:
Just use the normal `AFL_` env here? No reason to add a new one, IMHO.
Anyway, unicorn-specific env variables should start with `AFL_UNICORN` or maybe just `AFL_UC` to be consistent with AFL_QEMU, etc.

Reply:
why does it need to be an env? Using the `-c0` command line parameter should activate this (and set this env). this is how it works for qemu and frida mode too

Reply:
That enables cmplog but not compcov.

@Evian-Zhang could you elaborate the difference?

Reply:
Ah that must have slipped through review back then. Good time to change it, IMHO.

If you think it's already in use somewhere, you could still accept the old UNICORN_ env variable as well (and potentially print a depreciation warning).

The UNICORN_ variant was never mentioned in the docs, fwiw:
https://aflplus.plus/docs/env_variables/

Reply:
oh sorry, my bad, read CMPLOG. yes for COMPCOV please keep as is :-)